2024-10-05 13:48:06,671 Minimum price: 10, Maximum price: 350
2024-10-05 13:48:06,683 Preparing sklearn pipeline
2024-10-05 13:48:06,684 Fitting
Traceback (most recent call last):
  File "/root/Project-Build-an-ML-Pipeline-Starter/src/train_random_forest/run.py", line 301, in <module>
    go(args)
  File "/root/Project-Build-an-ML-Pipeline-Starter/src/train_random_forest/run.py", line 77, in go
    sk_pipe.fit(X_train, y_train)
  File "/root/miniconda3/envs/mlflow-cdefb483e70f2a40e681d53031fd66967657c2c2/lib/python3.9/site-packages/sklearn/base.py", line 1152, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/root/miniconda3/envs/mlflow-cdefb483e70f2a40e681d53031fd66967657c2c2/lib/python3.9/site-packages/sklearn/pipeline.py", line 427, in fit
    self._final_estimator.fit(Xt, y, **fit_params_last_step)
  File "/root/miniconda3/envs/mlflow-cdefb483e70f2a40e681d53031fd66967657c2c2/lib/python3.9/site-packages/sklearn/base.py", line 1145, in wrapper
    estimator._validate_params()
  File "/root/miniconda3/envs/mlflow-cdefb483e70f2a40e681d53031fd66967657c2c2/lib/python3.9/site-packages/sklearn/base.py", line 638, in _validate_params
    validate_parameter_constraints(
  File "/root/miniconda3/envs/mlflow-cdefb483e70f2a40e681d53031fd66967657c2c2/lib/python3.9/site-packages/sklearn/utils/_param_validation.py", line 96, in validate_parameter_constraints
    raise InvalidParameterError(
sklearn.utils._param_validation.InvalidParameterError: The 'criterion' parameter of RandomForestRegressor must be a str among {'friedman_mse', 'absolute_error', 'poisson', 'squared_error'}. Got 'mae' instead.